mmv1/templates/terraform/examples/workstation_cluster_private.tf.erb

```diff
@@ -0,0 +1,36 @@
+resource "google_workstations_workstation_cluster" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  workstation_cluster_id = "<%= ctx[:vars]['workstation_cluster_name'] %>"
+  location   = "us-central1"
+  network    = "projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}"
+  subnetwork = "projects/${data.google_project.project.number}/global/subnetworks/${google_compute_subnetwork.default.name}"
+
+  private_cluster_config {
+    enable_private_endpoint = true
+  }
+
+  labels = {
+    "label" = "key"
+  }
+
+  annotations = {
+    label-one = "value-one"
+  }
+}
+
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_compute_network" "default" {
+  provider = google-beta
+  name = "<%= ctx[:vars]['workstation_cluster_name'] %>"
+}
```

Comment:
You'll need to set `auto_create_subnetwork` to false here. Otherwise there is collision with the resource below.

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network#auto_create_subnetworks

Reply:
Fixed